rootfs/etc/nginx/template/nginx.tmpl

```diff
@@ -16,7 +16,9 @@ pid {{ .PID }};
 load_module /etc/nginx/modules/ngx_http_geoip2_module.so;
 {{ end }}
 
+{{ if $all.Cfg.EnableModsecurity }}
```

Comment:
This isn't enough because ModSecurity can be enabled per location too: https://github.com/kubernetes/ingress-nginx/blob/38ab2203331568d05eb7c587f1a4ff72e1a32623/rootfs/etc/nginx/template/nginx.tmpl#L1117

Reply:
Oh I missed your "special notes"

Reply:
What I said above still holds: https://github.com/kubernetes/ingress-nginx/pull/4119#issuecomment-495955005